testsuite/python-imagespec/ref/out.txt

```diff
@@ -74,20 +74,20 @@ channelformat(1) = uint8
 get_int_attribute('foo_int') retrieves 14
 get_int_attribute('foo_int',21) with default retrieves 14
 get_int_attribute('foo_no',23) retrieves 23
-get_float_attribute('foo_float') retrieves 3.1400001049
+get_float_attribute('foo_float') retrieves 3.140000104904175
 get_float_attribute('foo_float_no') retrieves 0.0
-get_float_attribute('foo_float_no',2.7) retrieves 2.70000004768
+get_float_attribute('foo_float_no',2.7) retrieves 2.700000047683716
```

Comment:
Why did these change?

Reply:
I suspect this is an artifact of the weirdo merge, maybe pulling in some things from other branches that you didn't intend.

Reply:
Mmmm.... unfortunately, I made these changes four months ago, and I forgot to say why exactly in the commit :(
I believe one or more of the MacOS runners were choking on the precision differences; but it's definitely possible the tests were only failing on my local machine, and these were changes I hadn't intended to commit... 

Should I revert these changes and see if stuff breaks?